pkg/kubelet/metrics/collectors/volume_stats.go

```diff
@@ -61,6 +61,12 @@ var (
 		[]string{"namespace", "persistentvolumeclaim"}, nil,
 		metrics.ALPHA, "",
 	)
+
+	volumeStatsHealthAbnormalDesc = metrics.NewDesc(
+		metrics.BuildFQName("", kubeletmetrics.KubeletSubsystem, kubeletmetrics.VolumeStatsHealthStatusKey),
+		"Volume health status. The count is either 1 or 0. 1 indicates the volume is unhealthy and 0 indicates volume is healthy",
```

Comment:
If the label `volume_health_status: healthy` and count is 1 - does it mean volume is unhealthy?

Reply:
The label volume_health_status="abnormal" 1 - the volume is unhealthy
The label volume_health_status="abnormal" 0 - the volume is healthy

Reply:
The doc comment should mention that if `volume_health_statu=abnormal` and count is - 1

Reply:
I changed the comment to `"Volume health status. The count is either 1 or 0. When volume_health_status is abnormal, 1 indicates the volume is unhealthy, 0 indicates volume is healthy"`.

I think `volume_health_status` value is precondition, it has larger scope than value of count.

Reply:
@gnufied Please review it again. Thanks ❤️